apps/web/src/components/utils/GotAQuestionButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { useIntercom } from 'react-use-intercom';
+
+import { INTERCOM_APP_ID } from '../../config';
+import { Button, Size } from '../../design-system';
+
+export function GotAQuestionButton({ mt, size }: { mt: number; size: Size }) {
+  if (!INTERCOM_APP_ID) {
+    return null;
+  }
+
+  const { boot } = useIntercom();
```

Comment:
```suggestion
  const { show } = useIntercom();
```

I think you can just call `show` since intercom at this point should already be booted